crm_secondary_salesperson/tests/test_crm_lead.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from psycopg2 import IntegrityError
+
+from odoo.tests import Form, common
+from odoo.tools import mute_logger
+
+
+class TestCrmProject(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user_admin = cls.env.ref("base.user_admin")
+        cls.user_demo = cls.env.ref("base.user_demo")
+
+    def test_crm_with_distinct_secondary_user_id(self):
+        lead = Form(self.env["crm.lead"])
+        lead.name = "Test"
+        lead.user_id = self.user_admin
+        lead.secondary_user_id = self.user_demo
+        lead = lead.save()
+        self.assertEqual(lead.secondary_user_id, self.user_demo)
+
+    def test_crm_with_secondary_user_id(self):
```

Comment:
What is the difference between this test and the previous one?

Reply:
Tests check 3 examples:
- Lead with user_id A + secondary_user_id B
- Lead without user_id + with secondary_user_id B
- Lead with user_id A + secondary_user_id A

Reply:
Yes, but when you don't assign a `user_id`, one is automatically assigned from context. In this case, you end up with `user_id` being the System User being used by the tests. 
In that case, does it really make sense to keep the 2 separate situations?